lib/tasks/compact_index.rake

```diff
@@ -91,9 +91,9 @@ namespace :compact_index do
   end
 
   desc "Generate/update the versions.list file"
-  task update_versions_file: :environment do
-    file_path = Rails.application.config.rubygems['versions_file_location']
-    versions_file = CompactIndex::VersionsFile.new file_path
+  task :update_versions_file, [:file_path] => :environment do |_t, args|
+    raise "#{args[:file_path]} does not exist" unless File.exist?(args[:file_path])
```

Comment:
I think we should not raise since this will often be a completely new file on disk... for this task, the file is written from scratch, not updating from an old file.